journal/journal_windows.go

```diff
@@ -33,3 +33,11 @@ func Enabled() bool {
 func Send(message string, priority Priority, vars map[string]string) error {
 	return errors.New("could not initialize socket to journald")
 }
+
+func StderrIsJournalStream() (bool, error) {
+	return false, nil
```

Comment:
This and the one below should probably return an error stating that they are not implemented.

Reply:
My idea of how it's going to be used is as follows:
```go
ok, err := journal.StderrIsJournalStream()
if err != nil {
    fmt.Fprintln(os.Stderr, "StderrIsJournalStream: %v", err)
}

if ok {
    // configure logger to use journal native protocol
} else {
    // configure logger to use stderr
}
```

In this case returning an error would create a useless noise on Windows. Or library users should be careful to suppress this specific error on Windows, both complicating their own code, not to mention that (I suppose) many Go developers rarely test their code on Windows at all.

Reply:
Fair point. I was expecting consumers to mostly merge together the `false` and `error` cases. I'm fine keeping this.